### PR TITLE
Bump spring boot version

### PR DESCRIPTION
**Pull request by wreulicke**
Upgrade spring boot release version.

### CODE REVIEW

**Comment by yvandecn** [comment]
:+1:

File: pom.xml [hunk 1]
```diff
@@ -565,7 +565,7 @@
             <id>spring-boot-2.0</id>
             <properties>
                 <spring.version>5.0.4.RELEASE</spring.version>
-                <spring-boot.version>2.0.0.RC2</spring-boot.version>
+                <spring-boot.version>2.0.0.RELEASE</spring-boot.version>
             </properties>
             <repositories>
```
Comment (whiskeysierra) [inline pom.xml]: Can you also remove the `repositories` section? We shouldn't need it anymore, since it's now in central.

**Comment by whiskeysierra** [comment]
:+1:

**Comment by whiskeysierra** [comment]
@wreulicke Thanks for the contribution! :tada:

**Pull request merged**